c7n/resources/dynamodb.py

```diff
@@ -120,6 +119,49 @@ def __call__(self, r):
         return super().__call__(r.get(self.annotation_key, {}))
 
 
+@Table.filter_registry.register('autoscaling')
+class DynamoDBAutoscalingFilter(ListItemFilter):
+    schema = type_schema(
+        'autoscaling',
+        attrs={'$ref': '#/definitions/filters_common/list_item_attrs'},
+        count={'type': 'number'},
+        count_op={'$ref': '#/definitions/filters_common/comparison_operators'}
+    )
+    permissions = ('application-autoscaling:DescribeScalableTargets',)
+    item_annotation_key = 'c7n:AutoscalingTargets'
+    annotate_items = True
+
+    def _process_resources_set(self, client, resources):
+        targets = client.describe_scalable_targets(
+            ServiceNamespace='dynamodb',
+            ResourceIds=[f'table/{t["TableName"]}' for t in resources]
```

Comment:
is there a limit on how many resource ids can be provided?

Reply:
Yes, max 50 scalable targets per call. I use chunks with 25 because one table can contain not more that 2 autoscaling targets, for read and write